composer.json

```diff
@@ -41,13 +41,15 @@
     "roave/security-advisories": "dev-master"
   },
   "require-dev": {
-    "dealerdirect/phpcodesniffer-composer-installer": "^0.4.3",
+    "php": "^5.6|^7",
+    "brain/monkey": "^2.2",
+    "dealerdirect/phpcodesniffer-composer-installer": "^0.4.4",
+    "mikey179/vfsStream": "^1.6",
+    "phpunit/phpunit": "~4.8 || ~5.7.9",
```

Comment:
If PHP 5.6 is minimum, then I think that means PHPUnit 4.* is out.

Reply:
You are correct. Commit coming....